uber/site_sections/accounts.py

```diff
@@ -20,8 +20,7 @@ def index(self, session, message=''):
                                .order_by(Attendee.first_name, Attendee.last_name).all(),
             'all_attendees': sorted([
                 (id, '{} - {}{}'.format(name.title(), c.BADGES[badge_type], ' #{}'.format(badge_num) if badge_num else ''))
-                for id, name, badge_type, badge_num in session.query(Attendee.id, Attendee.full_name, Attendee.badge_type, Attendee.badge_num)
-                                                              .filter(Attendee.email != '').all()
+                for id, name, badge_type, badge_num in session.query(Attendee.id, Attendee.full_name, Attendee.badge_type, Attendee.badge_num).all()
```

Comment:
The reason this was proposed is that we had a bunch of confused admins this year who kept trying to create another admin at regdesk, but they couldn't figure out why the attendee's name wasn't on the list.  The suggestion at the time was, "Let's put all of the names on the list, so they can select the person, and then they get told that the person doesn't have an email address so that people aren't confused."

There are probably better solutions, but this seems like an okay quick-and-easy thing to prevent that from happening.

Although come to think of it, we should probably prevent people without first/last names from showing up here.  So maybe include a ``.filter(Attendee.first_name != '')`` in the query, since otherwise we'll have tons and tons of unclaimed group badges showing up in the dropdown, which seems bad.